### DISCUSSION TITLE
What Integrations/Extensions/Plugins should we add?

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#185] [Ideas]
Let us know!

**Comment by rosie-at-pieces** [comment]
Microsoft teams

**Comment by rosie-at-pieces** [comment]
Bitbucket

**Comment by rosie-at-pieces** [comment]
Vim

**Reply by joshuadanpeterson** [reply]
And Neovim!

**Comment by rosie-at-pieces** [comment]
XCode

**Comment by rosie-at-pieces** [comment]
Zedd

**Reply by Avik-creator** [reply]
Yes Please

**Comment by apetta** [comment]
Raycast

**Reply by caleb-at-pieces** [reply]
Hey @apetta !

Happy to announce that we just released the Pieces Raycast Extension! https://www.raycast.com/pieces_dev/pieces-raycast

**Reply by apetta** [reply]
It's great! Thank you

**Comment by hra42** [comment]
Figma (for adding design context to wpe)

**Reply by rosie-at-pieces** [reply]
@HRA42 have you tried using it with the WPE so far? I used it with a Figma file the other day and it actually gave a good summary of what the design was. But I didn't try asking detailed questions so I'm curious what your experience is.

**Reply by hra42** [reply]
Yes @rosie-at-pieces you are right, it works quite well! Maybe this isn't needed, but my designs are really that complex.

Maybe some real designers have some thoughts.

**Comment by kapitanluffy** [comment]
Sublime Text!

**Reply by rosie-at-pieces** [reply]
@kapitanluffy are you interested in being a beta tester for our upcoming Sublime plugin??

**Reply by kapitanluffy** [reply]
Sure!

**Reply by rosie-at-pieces** [reply]
Amazing!! Check out this repo's readme to get started: https://github.com/pieces-app/plugin_sublime

And if you haven't already, please [join our Discord](https://discord.gg/getpieces)! Once you're in just DM me rosie-at-pieces and I'll add you to the beta testing channel. That's where we'd love to hear all of your feedback!

This is an open invite for anyone who wants to test out Sublime btw!

**Reply by kapitanluffy** [reply]
I messaged you on Discord! :D

**Reply by cole-at-pieces** [reply]
@kapitanluffy this plugin is now live! https://docs.pieces.app/extensions-plugins/sublime

**Comment by joshuadanpeterson** [comment]
Alfred

**Comment by GreatSmattering** [comment]
Structured text from and to natural language, and be able to convert one to the other, then reproduce in graphical ladder logic..

Im sure you already have it for visual studio code and teams?

Sorry for the difficult one!

**Comment by rmulligan** [comment]
Emacs please!

**Comment by polyglotdev** [comment]
[Ulysses App](https://ulysses.app/)

**Reply by hra42** [reply]
Are there extensions available? I couldn't find any examples on their website..

What could an integration solve that you miss right now?

**Comment by PiotrowskiD** [comment]
GitHub copilot

It works great but the context management is lacking + they try to limit it as much as possible to save costs. Creating meaningful context offline that could be (semi)automatically passed to prompts/conversations would be amazing

**Comment by leonormes** [comment]
Hi,
I am on MacOS and I use a context menu app called popclip. This is a link to the extensions page. https://www.popclip.app/extensions/
As you can see there are a lot of thinks that can be added, and it would be great to have a Pieces on as well. It is just a small integration but would make adding snippets even easier!

**Comment by aliabidhasan** [comment]
RStudio

**Comment by prudhvinath07** [comment]
zed editor 
https://zed.dev
